src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -10,6 +10,48 @@
     {{ parent() }}
 {% endblock %}
 
+{% block head_additions %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/monokai_sublime.min.css">
+    <script type="text/javascript">
+      WebFontConfig = {
+        google: { families: [ 'Source+Code+Pro::latin' ] }
+      };
+      (function() {
+        var wf = document.createElement('script');
+        wf.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js';
+        wf.type = 'text/javascript';
+        wf.async = 'true';
+        var s = document.getElementsByTagName('script')[0];
+        s.parentNode.insertBefore(wf, s);
+      })();
+      function highlight_code()
+      {
+        if (typeof (Worker) === undefined) return false;
+
+        var workerFunction = new Blob(['(' + highlight_code_worker_function.toString() + ')()'], {type: "text/javascript"});
+        var localWorkerURL = URL.createObjectURL(workerFunction);
+
+        $('div.readme pre, div.readme code').each(function() {
+          var code = $(this);
+          var worker = new Worker(localWorkerURL);
```

Comment:
I would not iterate over blocks inside workers. this would be hard (as you cannot pass DOM nodes to the worker).

what could be done is to start only 1 worker (when the ``$('div.readme pre, div.readme code')`` collection is not empty), and then  change a bit the worker integration. Instead of posting just the text, you would post a object with the text and an identifier. And ``onmessage`` would receive an object with 2 properties too (the highlighted HTML and the same identifier) and use the identifier to associate it to the right DOM node.
This way, you would keep a single worker, which would handle all messages (you should stop closing the worker at the end of the highlighting of course)